src/pages/[platform]/reference/amplify_outputs/index.mdx

```diff
@@ -35,6 +35,14 @@ You can also manually create this file for a specified Amplify app ID and branch
 
 {/* @TODO full type/interface to reference */}
 
+<InlineFilter filters={['react-native']}>
+<Callout>
+
+**Note:** if you are using Expo EAS build, you will need to force commit the `amplify_outputs.json` file to local git repository before running `eas build` command. This is because the `amplify_outputs.json` file is not included in the default `.gitignore`, and the `eas build` command will not include untracked files in the build.
```

Comment:
this will complicate git merges. Is there another way customers can set this up in EAS build? I'm not terribly familiar with it but it looks like folks can run arbitrary commands in the builds which would enable them to run the `generate outputs` command, assuming they have the ability to get credentials https://docs.expo.dev/eas/workflows/examples/#deploy-to-production-workflow

or environment variables?

Reply:
> force commit ... to local git repository

or is this instructing folks to commit it locally, only?

Reply:
Yes that's correct, only need to add the config file into a local repo, before running the build command, this file itself doesn't need to be pushed to remote. This is a lowest requirement, developers can use various methods to perform this step before running `eas build`.

Reply:
From my research the lowest requirement for EAS is to ensure all files required for building the app in the version control. [easignore](https://docs.expo.dev/build-reference/easignore/) is for ignoring files which is opposite to the requirement. [buildartifactpaths](https://docs.expo.dev/eas/json/#buildartifactpaths) is for post build steps I believe.

I will expand the callout.

Reply:
Updated.